Functions.Templates/ProjectTemplate_v4.x/CSharp-Isolated/Company.FunctionApp.csproj

```diff
@@ -10,10 +10,13 @@
     <RootNamespace Condition="'$(name)' != '$(name{-VALUE-FORMS-}safe_namespace)'">Company.FunctionApp</RootNamespace>
   </PropertyGroup>
   <ItemGroup>
+  <!--#if (NetCore)-->
+  	<FrameworkReference Include="Microsoft.AspNetCore.App" />
+  <!--#endif -->
     <PackageReference Include="Microsoft.Azure.Functions.Worker" Version="1.20.0" />
     <PackageReference Include="Microsoft.Azure.Functions.Worker.Sdk" Version="1.16.3" />
```

Comment:
https://www.nuget.org/packages/Microsoft.Azure.Functions.Worker.Sdk/1.16.4 is out now. Let's use that.